genai-perf/README.md

```diff
@@ -282,8 +282,8 @@ When the dataset is synthetic, you can specify the following options:
 
 When the dataset is coming from a file, you can specify the following
 options:
-* `--input-file <path>`: The input file containing the prompts to
-  use for benchmarking as JSON objects.
+* `--input-file <path>`: The input file or directory containing the prompts or 
+  images to use for benchmarking as JSON objects.
```

Comment:
are the images really json objects?

Reply:
Updated. Filepaths to images would be more accurate.